docs/Events/bpmn-events/bpmnElement-added.md

```diff
@@ -4,9 +4,33 @@ title: bpmnElement.added
 nav_order: 2
 description: bpmnElement.added event
 permalink: /events/bpmn-events/bpmnElement-added
-last_modified_date: 2020-04-27T17:54:08+0000
+last_modified_date: 2020-09-21T12:00:00+0000
 parent: "BPMN Events"
 grand_parent: "Events"
 ---
 
 # bpmnElement.added
+
+This event is fired by [bpmnImporter API](/modules/bpmn-modules/bpmn-importer) after an element was added to the [canvas](/modules/common-modules/canvas). This allows users to listen to whenever a BPMN 2.0 element got added during the import.
+
+## Example Usage
+
+```javascript
+// modeler is an instance of BpmnJS Modeler.
+
+modeler.on('bpmnElement.added', (data) => {
+
+  const { element } = data; // this is the element that was added to the canvas
+
+  if(element.businessObject.$type === 'bpmn:StartEvent') {
```

Comment:
If I remember correctly we had like this API called `is`. If that stuff is inteded to be used publicly, this might be a good place to use it as an example. Not crucial though.

Reply:
Not sure actually. It would need to import another dependency and would maybe blow up the code a bit. But I have no strong opinion on that.